src/server/services/toeristische-verhuur.ts

```diff
@@ -348,6 +348,9 @@ export function createToeristischeVerhuurNotification(
         cta = 'Bekijk uw aanvraag';
         linkTo = ctaLinkToDetail;
         datePublished = item.dateRequest;
+        if (item.caseType === CaseType.BBVergunning) {
+          datePublished = item.dateWorkflowActive || item.dateRequest;
+        }
         break;
       // B&B + Vakantieverhuurvergunning
       case item.status === 'Afgehandeld':
```

Comment:
Deze mag ook aangepast worden naar `!!item.decision` Bij de toeristische vergunningen vervangen we de status met een beslissing zie https://github.com/Amsterdam/mijn-amsterdam-frontend/blob/main/src/server/services/toeristische-verhuur.ts#L160-L169